src/main/java/worldofzuul/Rooms/House.java

```diff
@@ -5,8 +5,8 @@
 public class House extends Room{
     private int energyNeed;
 
-    public House(String description, int energyNeed){
-        super(description);
+    public House(String description, String name, int energyNeed){
```

Comment:
It feels better to put name first in the constructor, but it isn't a dealbreaker